app/portainer/settings/authentication/components/SaveAuthSettingsButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { LoadingButton } from '@/portainer/components/Button/LoadingButton';
+
+export interface Props {
+  onSaveSettings(): void;
+  saveButtonState: boolean;
```

Comment:
if I look at the code then it maps to `isLoading`, so why not just give it the same name?

Reply:
makes sense